.github/workflows/publish.yaml

```diff
@@ -0,0 +1,35 @@
+name: Publish Package to npmjs
+on:
+  push:
+    tags:
+      - v**
+      - test-ci-**
+  release:
+    types: [published]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      # Setup .npmrc file to publish to npm
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '20.x'
```

Comment:
Would recommend adding a `.nvmrc` file and using node v18 (eg v18.18.2) because it is the current LTS version. v20 is an active version that new features gets added to. You can then specify `node-version-file: .nvmrc` instead to have this version used.

The `nvm` tool uses `.nvmrc` files to install node versions in parallel.

(This applies to the other workflow too)

Reply:
done